.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: Cypress Tests
+
+on: push
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      
+      - name: Install npm dependencies
+        run: |
+          npm install
+          npm run install:frontend
+          npm run install:backend
+
+          # Uncommment this block of code if you want to run workflow changes locally using act: https://github.com/nektos/act
```

Comment:
I totally agree. Only reason I left it was because I knew I was gonna utilize it again when working to optmise my workflows, or else I'd not have left it :) All removed now.